### PR TITLE
Update models_quickstart.mdx

### PR DESCRIPTION
**Pull request by amysyk**
Replaced incorrect variable in sample code

## Description
Replaced `y_pred` with `loss` in `print(f"epoch={epoch}", loss = {y_pred})`

### CODE REVIEW

**Review by ngrayluna — APPROVED** [review]

**Comment by ngrayluna** [comment]
Thanks @amysyk !

**Comment by amysyk** [comment]
> Thanks @amysyk !

Thank you, [ngrayluna](https://github.com/ngrayluna). Mintlify checks stopped running after I rebased. Could you help me merge?

**Comment by mdlinville** [comment]
> > Thanks @amysyk !
> 
> Thank you, [ngrayluna](https://github.com/ngrayluna). Mintlify checks stopped running after I rebased. Could you help me merge?

It might be because you are using a fork. I'll go ahead and merge it and I'll follow up with Mintlify.
